.github/workflows/ci.yaml

```
@@ -300,17 +331,34 @@ jobs:
             exit 98
           }
 
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)